docs/source/_templates/breadcrumbs.html

```diff
@@ -0,0 +1,28 @@
+{%- extends "sphinx_rtd_theme/breadcrumbs.html" %}
+
+{% set display_vcs_links = display_vcs_links if display_vcs_links is defined else True %}
+
+{% block breadcrumbs_aside %}
+  <li class="wy-breadcrumbs-aside">
+    {% if hasdoc(pagename) and display_vcs_links %}
+      {% if display_github %}
+        {% if check_meta and 'github_url' in meta %}
+          <!-- User defined GitHub URL -->
+          <a href="{{ meta['github_url'] }}" class="fa fa-github"> {{ _('Edit on GitHub') }}</a>
+        {% else %}
+          {% if 'examples/index' in pagename %}
+            <a href="https://github.com/pyro-ppl/numpyro/blob/master/examples/README.rst" class="fa fa-github"> {{ _('Edit on GitHub') }}</a>
+          {% elif 'examples/' in pagename %}
+            <a href="https://github.com/pyro-ppl/numpyro/blob/master/{{ pagename }}.py" class="fa fa-github"> {{ _('Edit on GitHub') }}</a>
```

Comment:
I added the link to the example here. Codes below this are the default in sphinx-rtd-theme.